src/index.html

```diff
@@ -1,13 +1,408 @@
+
+            <a href="tel:+1 234 555-55-55" class="icon icon--phone header__call"></a>
+
+            <a
+              href="#menu"
+              class="icon icon--menu"
+            ></a>
+          </div>
+
+          <div class="header__title--position">
+            <h1 class="header__title">
+              Take the Streets
+            </h1>
+          </div>
+        </div>
+
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="images-bike/logo-black.png"
+                alt="MyBike"
+              >
+            </a>
+
+            <div></div>
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare bikes</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#details" class="menu__link">details</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+             href="tel:+1 234 555-55-55" class="menu__call menu__call--phone"
+          >
+            +1 234 555-55-55
+          </a>
+
+          <a
+            href="tel:+1 234 555-55-55" class="menu__call menu__call--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Move Free
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--7-12">
+
+              <p class="about__paragraph">
+                Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+              </p>
+
+              <!-- <p class="about__paragraph about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p> -->
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section compare-bikes" id="compare-bikes">
+        <div class="container">
+          <h2 class="page__section-title">
+            Compare Bikes
+          </h2>
```

Comment:
Price text should be on the left
![image](https://user-images.githubusercontent.com/37161315/170649675-89a9f4a6-7a50-4e59-b858-d528389f2b55.png)